lib/cib/cib_utils.c

```diff
@@ -184,38 +170,6 @@ element_in_patchset_v2(const xmlNode *patchset, const char *element)
     return rc;
 }
 
-/*!
- * \internal
- * \brief Check whether a given CIB element was modified in a CIB patchset
- *
- * \param[in] patchset  CIB XML patchset
- * \param[in] element   XML tag of CIB element to check (\c NULL is equivalent
- *                      to \c PCMK_XE_CIB). Supported values include any CIB
- *                      element supported by \c pcmk__cib_abs_xpath_for().
- *
- * \return \c true if \p element was modified, or \c false otherwise
- */
-bool
-cib__element_in_patchset(const xmlNode *patchset, const char *element)
-{
-    int format = 1;
-
-    CRM_ASSERT(patchset != NULL);
-
-    crm_element_value_int(patchset, PCMK_XA_FORMAT, &format);
-    switch (format) {
-        case 1:
-            return element_in_patchset_v1(patchset, element);
```

Comment:
this could affect remote nodes too

Reply:
What if anything is the implication for this PR of this and your other two comments about remote nodes? I'm not following

Reply:
Does T301 need to come before this?

Reply:
It would be part of that

Reply:
I meant before this PR, I was vague :)

Reply:
Yep, this PR would be part of T301 (iirc the only other thing needed is removing the pacemaker_remoted link)